examples/mobile/HomeApp/js/app.js

```diff
@@ -392,9 +430,7 @@ import Storage from "./clipping-storage.js";
 		tau.engine.createWidgets(webclipsContainer);
 	}
 
-	//TODO: provide mechanism for getting web clip name from webClip meta data
-	//		and separate from getting ID
-	function getWebClipName(webClip) {
+	function getWebClipNameFromUrl(webClip) {
```

Comment:
I think we should leave comment about getting webclipName from metadata and spearate it from getting webclip ID
we can have ID duplication problems in future

Reply:
ok